client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -133,12 +141,21 @@ button {
 		}
 	}
 
+	@include break-large {
+		height: 100%;
+	}
+
 	/**
  	 * Step Container
  	 */
 	.step-container {
 		--color-accent: #117ac9;
 		--color-accent-60: #0e64a5;
+
+		@media ( min-width: 660px ) {
```

Comment:
Is there any reason we need to specify the width rather instead of using something like `@include break-small`?